src/Data/Bool/Instances.agda

```diff
@@ -8,7 +8,7 @@
 
 module Data.Bool.Instances where
 
-open import Data.Bool.Properties
+open import Data.Bool.Properties using (≤-isDecTotalOrder; _≟_)
```

Comment:
Given that this choice gives rise to a merge conflict with `master`, following other PRs recently pushed, I think it's perverse to have these names in this order, when 
```suggestion
open import Data.Bool.Properties using (_≟_; ≤-isDecTotalOrder)
```
would avoid the merge conflict?

Reply:
Oh, oops, I didn't see these comments when I resolved the merge conflict and chose the other one! Too late now. Sorry, my mistake.